nixos/modules/installer/cd-dvd/installation-cd-base.nix

```diff
@@ -46,5 +46,5 @@ with lib;
     done
   '';
 
-  system.stateVersion = mkDefault "18.03";
+  system.stateVersion = lib.mkDefault lib.trivial.release;
```

Comment:
https://nixos.org/manual/nixos/unstable/options.html#opt-system.stateVersion says `config.system.nixos.release`, I guess that isn't meaningfully different?

Reply:
Pretty sure that will loop. Also this is more consistent with the test driver.

Reply:
Isn't that [already](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/misc/version.nix#L85) the default value?

Reply:
I think it'll still warn if it's not set.